examples/plot_objects_from_arrays.py

```diff
@@ -0,0 +1,121 @@
+"""
+=====================================
+Creating MNE objects from data arrays
+=====================================
+
+In this simple example, the creation of MNE objects from
+numpy arrays is demonstrated. In the last example case, a
+NEO file format is used as a source for the data.
+"""
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import mne
+
+import neo
+import numpy as np
```

Comment:
@jaeilepp we tend to import general libraries as numpy first, ....,  and mne last.